CRM/Core/Payment/Stripe.php

```diff
@@ -255,7 +255,7 @@ public function setAPIParams() {
     // Attempt one retry (Stripe default is 0) if we can't connect to Stripe servers
     Stripe::setMaxNetworkRetries(1);
     // Set plugin info and API credentials.
-    Stripe::setAppInfo('CiviCRM', CRM_Utils_System::version(), CRM_Utils_System::baseURL());
+    Stripe::setAppInfo('CanadaHelps', CRM_Utils_System::version(), CRM_Utils_System::baseURL());
```

Comment:
@Edzelopez this one I am not sure about, i'm not sure if this is the parameter that is used to determine if CiviCRM gets a commission or not @JoeMurray thoughts?

Reply:
I asked Matt about this. You were right, here is his response: "Stripe uses it to calculate the funding they provide to CiviCRM to support the core team"